transpiler/src/ast/core/BitloopsVisitor/BitloopsVisitor.ts

```diff
@@ -655,6 +656,17 @@ export default class BitloopsVisitor extends BitloopsParserVisitor {
     return readModelEvaluationVisitor(this, ctx);
   }
 
+  visitPackageEvaluation(ctx: BitloopsParser.PackageEvaluationContext): any {
+    return packageEvaluationVisitor(this, ctx);
+  }
+
+  visitPackageIdentifier(ctx: BitloopsParser.PackageIdentifierContext): IdentifierNode {
+    const commandName = ctx.PackageIdentifier().getText();
+    const metadata = produceMetadata(ctx, this);
+    const commandIdentifierNode = new IdentifierNodeBuilder(metadata).withName(commandName).build();
```

Comment:
little typo here command instead of package

Reply:
ah right, On my way!